src/main/java/uk/gov/gchq/palisade/exception/PalisadeCheckedException.java

```diff
@@ -52,6 +54,7 @@ public PalisadeCheckedException(final String message, final Throwable cause) {
         this(message, cause, INTERNAL_SERVER_ERROR);
     }
 
+    @Generated
     public Status getStatus() {
         return status;
     }
```

Comment:
Add toString, hashCode and equals methods

Reply:
See the comment for PalisadeCheckedException